templates/planka/index.ts

```diff
@@ -18,8 +12,12 @@ export function generate(input: Input): Output {
       env: [
         `TRUST_PROXY=0`,
         `BASE_URL=https://$(PRIMARY_DOMAIN)`,
-        `DATABASE_URL=postgres://postgres:${databasePassword}@$(PROJECT_NAME)_${input.databaseServiceName}:5432/$(PROJECT_NAME)?sslmode=disable`,
-        `SECRET_KEY=${secret}`,
+        `DATABASE_URL=postgresql://postgres:${databasePassword}@$(PROJECT_NAME)_${input.databaseServiceName}:5432/$(PROJECT_NAME)`,
+        `SECRET_KEY=nosecretkey`,
+        `DEFAULT_ADMIN_EMAIL=demo@demo.demo`,
```

Comment:
are these required? can't we allow the user to configure their admin account?

Reply:
The credentials were required. This is the official Doc: https://docs.planka.cloud/docs/default_admin/ - I have made the envs dynamic now.